pkg/commons-node/spec/which-spec.js

```diff
@@ -0,0 +1,77 @@
+'use babel';
+/* @flow */
+
+import which from '../which';
+
+describe('which', () => {
+  let checkOutput: JasmineSpy;
+  let checkOutputReturn = {stdout: ''};
```

Comment:
One more nit -- don't set a default value here, because if one test overrides it and another doesn't, the second one will end up with the value from the first test which is really confusing. Instead, set it to `null` here (one trick to satisfy Flow is to do something like `let checkOutputReturn: TypeAnnotation = (null: any)`), then set the default value at the top of the `beforeEach`.